blog: November newsletter

orrabin opened the pull request.

orrabin committed:
blog: November newsletter

GregSutcliffe commented on _posts/2017-11-30-foreman-community-newsletter---november-2017.md:
```diff
@@ -0,0 +1,94 @@
+
+We had a busy demo with changes all around - the installer, smart proxies, UI.
+This included two introduction to new proposals - [Foreman telemetry proposal]
+and discourse.
```
Capital D for Discourse :P

GregSutcliffe commented:
Looks good, I added a Discourse paragraph. I guess 1.16 will be out today, so lets wait a little bit before merging this.

ekohl commented on _posts/2017-11-30-foreman-community-newsletter---november-2017.md:
```diff
@@ -0,0 +1,94 @@
+Discourse in the near future, so watch out for more announcements about
+that in the coming weeks.
+
+### Katello 3.5 RC2
```
I think that @akofink is working on the final release. Also note that @dLobatog is working on building the foreman 1.16.0 release. might be good to consider that too.

akofink replied:
Yes, it's quite impossible (afaik) to build plugin releases (i.e. Katello) without having built Foreman first; however, out goal is to release 3.5 GA as quickly as possible after Foreman's GA.

orrabin commented:
@GregSutcliffe, @ekohl I added 1.16 and a comment about katello 3.5 upcoming release.

GregSutcliffe commented on _posts/2017-11-30-foreman-community-newsletter---november-2017.md:
```diff
@@ -0,0 +1,105 @@
+
+### New Sponsors?
+
+Fastly, our new sponsor has provided us with a [cdn] for the website.
```
Was `[cdn]` meant to be a link?

ekohl replied:
This is actually a link. If the names match this is sufficient. So you could rename k_issues at the bottom to ```Katello bug tracker``` and simplify that link if you want.

orrabin replied:
It was but I can change it.

GregSutcliffe commented on _posts/2017-11-30-foreman-community-newsletter---november-2017.md:
```diff
@@ -0,0 +1,105 @@
+
+### Upcoming Events
+
+* Community Demo - 7th December  (URL to be announced)
```
Add Walden's UX demo?

GregSutcliffe commented:
Two small comments, then +1 @orrabin

orrabin commented:
@GregSutcliffe I removed the link from cdn, I don't think we actually need it.
I also added a link to the demo since it's next week and I have the link, and the UX review that I missed.

orrabin commented:
@GregSutcliffe approved this before leaving, no more comments were added so this can be merged.

tbrisker commented on _posts/2017-11-30-foreman-community-newsletter---november-2017.md:
```diff
@@ -0,0 +1,112 @@
+Until the beginning of May please feel free to contact Ori Rabin,
+orabin on irc.
+
+### New Sponsors?
```
I don't think you need the `?` here.

tbrisker commented on _posts/2017-11-30-foreman-community-newsletter---november-2017.md:
```diff
@@ -0,0 +1,112 @@
+### New Sponsors?
+
+Fastly, our new sponsor has provided us with a new CDN for the website.
+
```
extra blank line

tbrisker reviewed (changes requested):
a couple of tiny comments

orrabin commented:
@tbrisker thanks, fixed.

tbrisker commented:
Thanks @orrabin !
